_includes/_head.html

```diff
@@ -22,7 +22,7 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<title>{{ title }}</title>
 
-    <link rel="stylesheet" type="text/css" href="{{ url }}/assets/css/styles_feeling_responsive.css">
+    <link rel="stylesheet" type="text/css" href="assets/css/styles_feeling_responsive.css">
```

Comment:
Same point: I don't think there is any advantage in replacing the absolute URL with a relative one